src/azure-cli/azure/cli/command_modules/vm/_help.py

```diff
@@ -603,7 +608,7 @@
 examples:
   - name: Update a snapshot. (autogenerated)
     text: |
-        az snapshot update --name MySnapshot --resource-group MyResourceGroup --subscription MySubscription
```

Comment:
Some changes add `--subscription MySubscription`
Try to be consistent.

Reply:
added